src/main/java/io/cdap/plugin/http/source/batch/HttpBatchSourceConfig.java

```diff
@@ -123,6 +119,10 @@ public void validateBasicAuthResponse(FailureCollector collector, HttpClient htt
           collector.addFailure(errorMessage, "Please ensure that correct credentials are provided.");
         }
       }
+    } catch (IOException e) {
```

Comment:
do we need catch block for HttpHostConnectException which was there previously

Reply:
added back